Makefile

```diff
@@ -118,6 +118,24 @@ docker-build-dev: check-env-docker-repo  git-commit-sha
 	docker build --build-arg=GIT_COMMIT=$(GIT_COMMIT) -t $(DOCKER_REGISTRY_SERVER)/$(OPERATOR_IMAGE):$(GIT_COMMIT) .
 	docker push $(DOCKER_REGISTRY_SERVER)/$(OPERATOR_IMAGE):$(GIT_COMMIT)
 
+CERT_MANAGER_VERSION ?= 1.2.0
+CERT_MANAGER_HELM_RELEASE := cert-manager
+CERT_MANAGER_NAMESPACE := cert-manager
+cert-manager:
+	@echo "Installing Cert Manager"
+	helm repo add jetstack https://charts.jetstack.io
+	helm upgrade $(CERT_MANAGER_HELM_RELEASE) jetstack/$(@) \
+		--install \
+		--namespace $(CERT_MANAGER_NAMESPACE) --create-namespace \
+		--version $(CERT_MANAGER_VERSION) \
+		--wait
+
+cert-manager-rm:
```

Comment:
Is it worth having `helm repo rm jetstack` in this target?

Reply:
Actually, yes, it is worth 😄